lib/edit-site-page.php

```diff
@@ -189,6 +189,7 @@ function gutenberg_edit_site_init( $hook ) {
 		$template_part_ids = $template_part_ids + $_wp_current_template_part_ids;
 	}
 	$settings['templateId']      = $current_template_post->ID;
+	$settings['homeTemplateId']  = $current_template_post->ID;
```

Comment:
Is this meant to grab the template that is assigned as the home every time?

Reply:
Yes

Reply:
I'm confused then regarding when `templateId` and `homeTemplateId` would be different.

Reply:
When the client changes `templateId` to switch templates.